plugins/pulp_rpm/plugins/importers/yum/pulp_solv.py

```diff
@@ -284,10 +217,20 @@ def rpm_dependency_conversion(solvable, unit_name, unit_flags, unit_evr,
 
     :param solvable: a libsolv solvable object
     :type solv.Solvable: a libsolv solvable
+
     :param unit: the content unit to get the dependencies from
     :type unit: an object or a dictionary
+
     :returns: None
     """
+
+    unit_name = unit.get('name', None)
```

Comment:
I believe `get` returns `None` by default if key is not found.
It looks more redundant to me, rather than just explicit (if that was the purpose). It's subjective so entirely up to you.
My only ask is to make it consistent. Throughout the code there are places where `None` is specified in the `get`  and there places where it is not.

Is it possible for unit to be an object which has a custom `get` method and not a standard one from the `dict`?

Reply:
The type of ```unit``` is actually a mongodb object that acts like a dict and possibly inherits from dict (didn't check).

It's probably redundant and unnecessary, but since that's the way it was written in the decorator, I copied and pasted the same pattern everywhere.  I'll make a temporary commit changing them to be bare and then rerun tests.